app/presentation/RoomItem/LastMessage.js

```diff
@@ -35,7 +35,7 @@ const formatMsg = ({
 		prefix = `${ lastMessage.u.username }: `;
 	}
 
-	let msg = `${ prefix }${ lastMessage.msg.replace(/[\n\t\r]/igm, '') }`;
+	let msg = `${ prefix }${ lastMessage.msg.replace(/[\n\t\r]/igm, '').trim() }`;
```

Comment:
What are you doing here?

Reply:
it was just a unsuccessfully tried to remove the two white spaces in markdown message preview that i forgot to remove.